,

```diff
@@ -1,4 +1,4 @@
-RSpec.describe StatusUpdateWorker do
```

Comment:
This file seems to have been renamed to `,` 😬

Reply:
Oh yeah, I remember forgetting to delete the comma file. (Like a dotfile, but more suspensful.) Updated.